.github/workflows/docker-images.yaml

```diff
@@ -22,19 +22,29 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        flavor:
-          - ""
-          - "-conda"
-          - "-kubernetes"
+        image:
+          - "prefecthq/prefect"
+          - "prefecthq/prefect-client"
+        release:
+          - true
+          - false
```

Comment:
Why did you add this? We determine which images to push based on how the pipeline was triggered, so I'm not sure this is necessary.

Reply:
Ahh, okay that was from how I thought the matrix worked, but you pointed me in the right direction with `exclude` and I think you're right, we don't need that.